config/crd/openstackproviderconfig_v1alpha1.yaml

```diff
@@ -37,7 +37,7 @@ spec:
           items:
             type: string
           type: array
-        root_volume:
+        rootVolume:
```

Comment:
Could you change and add the fields here as well?

In the future, I want us to finally migrate to `kubebuilder` autogeneration of this file. But we are not there yet.